addons/mail/static/tests/helpers/mock_server/models/mail_tracking_value.js

```diff
@@ -105,30 +103,38 @@ patch(MockServer.prototype, "mail/models/mail_tracking_value", {
      * Simulates `_tracking_value_format` on `mail.tracking.value`
      */
     _mockMailTrackingValue_TrackingValueFormat(tracking_value_ids) {
-        const trackingValues = tracking_value_ids.map((tracking) => ({
-            changedField: tracking.field_desc,
-            id: tracking.id,
-            newValue: {
-                fieldType: tracking.field_type,
-                value: this._mockMailTrackingValue_GetDisplayValue(tracking, "new"),
-            },
-            oldValue: {
-                fieldType: tracking.field_type,
-                value: this._mockMailTrackingValue_GetDisplayValue(tracking, "old"),
-            },
-        }));
+        const trackingValues = tracking_value_ids.map((tracking) => {
+            const irField = this.models["ir.model.fields"].records.find(
+                (field) => field.id === tracking.field_id
+            );
+            return {
+                changedField: irField.ttype.charAt(0).toUpperCase() + irField.ttype.slice(1),
+                id: tracking.id,
+                newValue: {
+                    fieldType: irField.ttype,
```

Comment:
note: while you are at it and I see the diff, it would make sense to change the formatter (also in python) to return the fieldType top level once, rather than on each new/old value separately